### DISCUSSION TITLE
How to center slides using embla carousel?

### DISCUSSION THREAD

**Discussion by siamahnaf** [davidjerleke/embla-carousel#452] [Help] [answered]
Hello, I use embla carousel like this-

```
import Image from "next/image";
import useEmblaCarousel from "embla-carousel-react";

//Data
import GalleryData from "@/Data/Gallery.data";

const Carousel = () => {
    const [emblaRef] = useEmblaCarousel({
        align: "center",
        containScroll: "trimSnaps",
        skipSnaps: true
    })
    return (
        <div className="embla overflow-hidden mt-8">
            <div className="embla__viewport" ref={emblaRef}>
                <div className="embla__container flex gap-3">
                    {GalleryData.map((item, i) => (
                        <div
                            className="embla__slide flex-[0_0_25%]"
                            key={i}
                        >
                            <div className={`${i % 2 !== 0 ? "mt-4" : ""}`}>
                                <Image src={item.image1} alt={item.alt1} width={418} height={322} className="mb-3" />
                                <Image src={item.image2} alt={item.alt2} width={418} height={322} />
                            </div>
                        </div>
                    ))}
                </div>
            </div>
        </div>
    );
};

export default Carousel;
```

Here I use TailwindCss-

It shows result like this-

![Screenshot (1)](https://user-images.githubusercontent.com/78312010/230891737-7223c671-cea8-42f6-b906-1f6120692e15.png)

But I need to show it initally like this-

![Screenshot (2)](https://user-images.githubusercontent.com/78312010/230891795-a4c5389e-3722-4ebd-bea0-8e0f4801bd85.png)

**Comment by fabz1001** [comment]
Hi @siamahnaf,

It's always best to provide a codesandbox for a better and faster help, however, I think I may have spotted the problem. 
You need to play with 2 more Embla options in order to achieve the desired result: [`slidesToScroll`](https://www.embla-carousel.com/api/options/#slidestoscroll) and [`startIndex`](https://www.embla-carousel.com/api/options/#startindex). 

Please give them a try and let us know if it helped. Thanks!

**Reply by davidjerleke** [reply, marked as answer]
@fabz1001 thanks for the help!

> It's always best to provide a codesandbox for a better and faster help

@siamahnaf This is very true. I prioritise questions with a CodeSandbox which is stated in the [contribution guidelines](https://github.com/davidjerleke/embla-carousel/blob/master/.github/CONTRIBUTING.md#bug-reports).

You need to set [`startIndex`](https://www.embla-carousel.com/api/options/#startindex) to `2` to achieve what you want.

**Reply by siamahnaf** [reply]
Yes, I can understand and it's working for me. Thank you very much.
